pubspec.yaml

```diff
@@ -140,7 +143,9 @@ flutter:
     - packages/smartcontracts/contracts/apps/SimpleFaucet.abi.json
     - packages/smartcontracts/contracts/standards/ERC20.abi.json
     - packages/smartcontracts/contracts/apps/Profile.abi.json
-  uses-material-design: false
+   
+  uses-material-design: true
```

Comment:
why do we need to enable material design? this will add additional libraries for no reason.

Reply:
this only affects including material icons in the app. We don't use material icons.

Reply:
removed